### ISSUE TITLE
Record added to child row but data not showing; using input

### ISSUE THREAD

**Issue by amberdiehl** [hikalkan/jtable#1995]
Hi, me again!

I am now seeing the child record added to my jTable but the value that is returned for the record is not rendering. In the parent table the exact same setup works.

In order to add validation (i.e. restrict field length, and add an asterisk next to required fields) I am using the 'input' function to render specific fields. In the parent, this seems to work fine; meaning, I create a new record and the returned group name is retuned and displayed for the new row in the table. 

When I use this same setup for the child table(s), the row is returned/setup in the child table, but the value for the field is rendered blank. In the example screen shots below, the first is directly after adding the new category. The second is when I refresh the page; it shows the category was added and now displays it.

The data is being stored in the database and I have validate that it is being returned in the JSON value.

Any idea how to solve this? Thanks in advance!
<img width="832" alt="screen shot 2016-10-07 at 7 36 58 pm" src="https://cloud.githubusercontent.com/assets/5828080/19209800/1b08b7e8-8cc7-11e6-8760-f6132e7ac540.png">
<img width="839" alt="screen shot 2016-10-07 at 7 37 44 pm" src="https://cloud.githubusercontent.com/assets/5828080/19209801/1b2963c6-8cc7-11e6-9c0c-1524e5dfb97d.png">
<img width="995" alt="screen shot 2016-10-07 at 7 38 43 pm" src="https://cloud.githubusercontent.com/assets/5828080/19209806/4bfe2fcc-8cc7-11e6-867d-ca034399ddf8.png">

**Comment by sheryever** [comment]
There is nothing wrong with the code. Rendering is based on your json data and `display` function of fieled.

Check your json data which you are returning from the server after adding the record.
May be there will be field name issue.

**Comment by amberdiehl** [comment]
Hi sheryever,

Thank you very much for your kind response. You were, of course, correct that the issue was how the data was being returned. 

For others who might experience this I thought I would follow up. I was return the data when adding a single record like this (I had copied the code for records, where they're returned as an array):

{"Result": "OK", "Record": **[**{"id": 00, "field1": "data"}**]**}

When I removed the brackets the field value is rendered in the row.
